test/ClangImporter/cfuncs_parse.swift

```diff
@@ -68,9 +68,9 @@ func test_pow() {
 }
 
 // https://github.com/apple/swift/issues/51573
-// long doubles in AAPCS64 are 128 bits, which is not supported by
-// Swift, so don't test this.
-#if !((os(Android) || os(Linux)) && arch(arm64))
+// long doubles in AAPCS64 and 64-bit Android are 128 bits, which is not
+// supported by Swift, so don't test this.
+#if !((os(Android) && _pointerBitWidth(_64)) || (os(Linux) && arch(arm64)))
```

Comment:
This seems wrong. I believe that Android X64 will not do FP128. I think that you should do `arch(arm64)` here.

Reply:
[Android x86_64 `long double` is 128-bit also](https://reviews.llvm.org/D82152). The precise error is that it [cannot register this `powl()` invocation on the Android x86_64 CI](https://ci-external.swift.org/job/oss-swift-RA-linux-ubuntu-24.04-android-build/439/consoleText):
```
ClangImporter/cfuncs_parse.swift:75:3: error: unexpected error produced: cannot find 'powl' in scope
  powl(1.5, 2.5)
  ^
```
This change got this test to pass locally for me.